README.md

```diff
@@ -1,3 +1,46 @@
-# Jellyfin Chromecast
+<h1 align="center">Jellyfin Chromecast Web Receiver</h1>
+<h3 align="center">Part of the <a href="https://jellyfin.org">Jellyfin Project</a></h3>
 
-Source code for the Jellyfin Chromecast receiver application
+<p align="center">
+<img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
+<br/>
+<br/>
+<a href="https://github.com/jellyfin/jellyfin">
+<img alt="GPL 2.0 License" src="https://img.shields.io/github/license/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+<a href="https://github.com/jellyfin/jellyfin/releases">
+<img alt="Current Release" src="https://img.shields.io/github/release/jellyfin/jellyfin-chromecast.svg"/>
+</a>
+</p>
+
+Jellyfin Chromecast is a receiver app used when casting to a Google Cast capable device. It is used when casting from the Jellyfin Android client or Jellyfin web client.
+
+### How do I use it?
+
+A `stable` and `unstable` version of this app are already included in `jellyfin-server`. There is no need to seperately install this project. To host your own version (for developing) see `CONTRIBUTING.md`.
+
+### What does it do?
+
+This is a `web receiver` as defined in the [Google Chromecast architecture](https://developers.google.com/cast/docs/overview).
+
+As soon as you push the "cast" button on your client this application will start on you cast-capable device & handle it from there. 
+
+### What doesn't it do?
+
+Anything related to your non-cast device (e.g. your phone, browser, other device) or anything about the inclusion of casting for a specific client (e.g. casting from the iOS app).
+
+Any issues/features related to that: check the respective repository.
+
+### Something not working right?
+First check if the issue is actually Google Cast related. So answer the question:
+
+`"Can I reproduce the issue on any other device than a Google Chromecast?"`
```

Comment:
```suggestion
`"Can I reproduce the issue on any other device than a Google Chromecast?"`
```

My Android TV contains Chromecast functionality but it is not a Google Chromecast. Not sure how to rephrase it though.

Reply:
Oof... I see what you mean. Additionally: you can both cast to the android tv and run the android tv jellyfin client. (And only the first one is relevant for this repo). I've changed the wording. But open to other suggestions.